en/docs/tutorials/integrating-with-solace.md

```diff
@@ -0,0 +1,144 @@
+3. You will need the following information to integrate the broker with API Manager
+    1. Username
+    2. Password
+    3. Developer name
+    4. Organization name
+4. Open the `<API-M_HOME>/repository/conf/deployment.toml` file and add the following according to your account details. Make sure to restart the server after adding the configurations. 
+    ```toml
+        [[apim.gateway.environment]]
+        name = <Name of the Solace Environment>
+        display_name = <Display Name of the Solace Environment>
+        type = <Type of the Environment>
+        display_in_api_console = <Displays the environment under 'Try it' in the API Developer Portal, in the API console>
+        description = <Description about Solace Environment>
+        provider = <Vendor name of the environment. (Must be "solace")>
+        service_url = <Base URL of the Solace broker>
+        username = <Username of the Solace broker account>
+        password = <Password of the Solace broker account>
+        ws_endpoint = <WS Endpoint of the Solace broker environment>
+        wss_endpoint = <WSS Endpoint of the Solace broker environment>
+        http_endpoint = <HTTP Endpoint of the Solace broker environment>
+        https_endpoint = <HTTPS Endpoint of the Solace broker environment>
+        show_as_token_endpoint_url = false
+
+        [apim.gateway.environment.properties]
+        Organization = <Password of the Solace broker account>
+        DevAccountName = <Password of the Solace broker account>
+    ```
+
+5. The Solace broker gateway is now available in the API Publisher.
+    <a href="{{base_path}}/assets/img/tutorials/solace/view-solace-broker.png"><img src="{{base_path}}/assets/img/tutorials/solace/view-solace-broker.png" alt="View Solace Broker" name="View Solace Broker"></a>
+
+## Creating and deploying an API
+
+You can use the Async API specification generated from the Solace PubSub+ Hub to create APIs in WSO2 API Manager.
+
+### Step 1 - Importing the Async API from Solace
+
+Before you begin: Make sure that you have integrated the Solace event broker with WSO2 API Manager as shown in the above section.
+
+1. Login to the API Publisher via **https://&lt;hostname>:9443/publisher**
+2. Click **CREATE API** and then click **Import AsyncAPI Definition**
+    <a href="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png"><img src="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png" alt="Import AsyncAPI Definition" name="Import AsyncAPI Definition"></a>
+
+3. The following two options to import the AsynAPI definition appear.
+    1. AsyncAPI URL - If you select this option, you need to provide a URL of a Solace API Specification.
+    2. AsyncAPI File - If you select this option, click Browse File to Upload and upload a file, which contains a Solace AsyncAPI definition.
+    Select your preferred method and click **NEXT**.
+
+4. Edit the Solace Async API information and click **Create**.
+
```

Comment:
Shall we add the screenshot to this step to show the Create option with identified solace protocols?

Reply:
@ruthryi Shall we add the ss for this?